Files/Filesystem/FilesystemOperations/Helpers/FilesystemHelpers.cs

```diff
@@ -707,11 +708,24 @@ public async Task<ReturnResult> RenameAsync(IStorageItemWithPath source, string
             Progress<FileSystemStatusCode> errorCode = new Progress<FileSystemStatusCode>();
             errorCode.ProgressChanged += (s, e) => returnCode = e;
 
-            IStorageHistory history = await filesystemOperations.RenameAsync(source, newName, collision, errorCode, cancellationToken);
+            if(source.ItemType == FilesystemItemType.File && Path.GetExtension(source.Path) != Path.GetExtension(newName))
+			{
+                var dialog = new MessageDialog("If you change a file extension, the file might become unusable." 
```

Comment:
Can you add strings for the new dialog to the en-US resource file?

Reply:
Sure happy to add the text to the en-US file. 

Can you confirm if you want me to access the string programatically or do you want me to add this popup modal as Xaml to the Widgets folder? I don't want to add manual calls in code to "ResourceLoader()" if this is not desired.

Reply:
@jusso-dev Looks good, thank you!